### ISSUE TITLE
Hubble 2 not accessible in skirmish mode

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/open-ig#720]
```
Game version: open-ig-0.95.129


Operating System: Windows 7 x64


Java runtime version: 1.7.0_11


Installed using the Launcher?  yes


Game language (en, hu, de): hu, en


What steps will reproduce the problem?
1. Skirmish mode
2. Tech level 6
3. Add players
4. Start game
5. Production screen, satellites
6. The "Orbital radar" is named as "Space telescope", and after it is 
manufactured, hubble 2 deployment is still not possible. Hubble 2 is entirely 
unaccessable.

What is the expected output? What do you see instead?


Please provide any additional information below.


Please upload any save before and/or after the problem happened. Please
attach the open-ig.log file found in the
application's directory.
```

Original issue reported on code.google.com by `tokfozel...@gmail.com` on 30 Mar 2013 at 8:05

**Comment by GoogleCodeExporter** [comment]
```
This only appears at Tech level 6
```

Original comment by `tokfozel...@gmail.com` on 30 Mar 2013 at 8:07

**Comment by GoogleCodeExporter** [comment]
```
Hello and thanks for reporting. Level 6 is a bit tricky because I store 
campaign-scripted tech under it, but I fixed the issue in 0.95.130 available 
now. Please note that currently there is no way for deploying space telescope 
in skirmish.
```

Original comment by `akarn...@gmail.com` on 30 Mar 2013 at 9:21
- Changed state: **Fixed**
- Added labels: **Technology**
